### ISSUE TITLE
Allow "bundled charts" to be easily configurable in Rancher helm chart

### ISSUE THREAD

**Issue by deniseschannon** [rancher/rancher#22823]
In 2.3, we've introduced the environment variable `CATTLE_SYSTEM_CATALOG=bundled` in order to use the packaged system charts. We should update the helm chart so it's an easy configuration option to set.

For now, let's plan to have the helm chart option be something like: 
```
--set useBundledSystemChart=true
```

**Comment by guangbochen** [comment]
@deniseschannon the default value of `useBundledSystemChart` should be `true`, correct?

**Comment by guangbochen** [comment]
link PR to https://github.com/rancher/rancher/pull/22825.

**Comment by jiaqiluo** [comment]
The bug fix is validated in Rancher:v2.3.0-rc5 in the air-gapped environment

- Render the template with the option ` --set useBundledSystemChart=true`

```
helm template ./rancher-2.3.0-rc5.tgz --output-dir .  \
 --name rancher \
 --namespace cattle-system \
 --set hostname=my-hostname \
 --set rancherImage=my-registry/rancher/rancher \
 --set systemDefaultRegistry=my-registry \
 --set useBundledSystemChart=true
```

- follow the documentation to install Rancher in airgap HA mode 
- add a custom cluster 
- enable the cluster monitoring 


Result:
- the `system-chart` is hidden from the UI
- the cluster monitoring is deployed successfully and works as expected 

<img width="1488" alt="Screen Shot 2019-09-18 at 7 25 47 PM" src="https://user-images.githubusercontent.com/6218999/65208504-313c9600-da4a-11e9-92c7-cf8eb56618ef.png">
